tests/Domain/Package/PackageManagerTest.php

```diff
@@ -38,6 +38,7 @@ class PackageManagerTest extends TestCase {
         '*' => ['version' => '~1.0', 'version_dev' => '1.x-dev'],
       ],
     ],
+    'drupal/module6' => ['type' => 'composer-plugin'],
```

Comment:
Not to be pedantic, but if it's a Composer plugin, it can't be a Drupal module, right?

```suggestion
    'project/package' => ['type' => 'composer-plugin'],
```

Reply:
Yes, correct.